gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/RawResourceController.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Coveo, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.gate.controllers;
+
+import com.netflix.spinnaker.gate.services.RawResourceService;
+import groovy.transform.CompileStatic;
+import io.swagger.annotations.ApiOperation;
+import java.util.List;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@CompileStatic
+@RestController
+public class RawResourceController {
+
+  @Autowired RawResourceService rawResourceService;
+
+  @ApiOperation(
+      value = "Retrieve a list of raw resources for a given application",
+      response = List.class)
+  @RequestMapping(value = "/applications/{application}/rawResources", method = RequestMethod.GET)
+  List<Map> getApplicationRawResources(
+      @PathVariable String application,
+      @RequestHeader(value = "X-RateLimit-App", required = false) String sourceApp) {
```

Comment:
Not sure this header value is actually used anywhere downstream of this call?

Reply:
Ah my bad. I saw it in other locations where it was set but not used so I assumed there was some Java/Spring magic that considered it for all endpoints

Reply:
Removed it